haproxy/state/upstream.go

```diff
@@ -56,12 +56,12 @@ func generateUpstream(opts Options, certStore CertificateStore, cfg consul.Upstr
 			Mode: beMode,
 		},
 	}
-	if opts.LogRequests && opts.LogSocket != "" {
+	if opts.LogRequests && opts.LogAddress != "" {
 		be.LogTarget = &models.LogTarget{
 			ID:       int64p(0),
-			Address:  opts.LogSocket,
+			Address:  opts.LogAddress,
 			Facility: models.LogTargetFacilityLocal0,
-			Format:   models.LogTargetFormatRfc5424,
+			Format:   models.LogTargetFormatRfc3164,
```

Comment:
Why change the format here? Was it tested with the "in-app" syslog client ?

Reply:
Sorry, that's the log format I was using elsewhere and I thought it's the more common one. It did not work with the "in-app" syslog client - reverted.